_pages/en/esdc-it-strategy.md

```diff
@@ -52,43 +47,125 @@ But a complex and uncertain world rewards an empirical cycle of trying, observin
+
+**2) IITB becoming a generative type organization**
+
+An organization culture that is high-trust and emphasizes information flow between teams is necessary to achieve excellence.
+The [Westrum organization topology model](https://cloud.google.com/solutions/devops/devops-culture-westrum-organizational-culture), by Dr. Ron Westrum, will be used as a guide to move towards and measure the organizational culture as follows:
+
+- The cooperation level between teams
+- How the reception of bad news is treated
+- The level at which shared responsibilities are encouraged and observed
+- The level at which bridging, breaking down silos, is encouraged and observed
+- How the response to failure is managed
+- The level at which experimentation is encouraged and observed
 
-The approach to the IT Strategy is **not** to complete each of the diagram's block before publishing them as a single plan.
-Instead, the ESDC IT Strategy team will work with the IT community to continuously diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours.
+**3) Reduced gap between the user and IT**
+
+Moving towards an empirical cycle of trying, observing, and correcting when managing IT initiatives will require involving the user throughout the exercise.
+This involves:
+
+- Increased focus on outcomes over advance planning activities
+- Increasing the frequency of deployment in production to receive feedback from end-users
+- Allowing IT teams to engage directly with end-users
+- Increased awareness of policy makers on the impacts of policies to end-users and IT systems
+
+<!--markdownlint-enable MD029 MD036-->
+
+### Long Term Outcomes (Impact)
+
+The long term impacts of the IT Strategy is to create the following two conditions:
+
+<!--markdownlint-disable MD029 MD036-->
+**1) Technology is a shared accountability between stakeholders**
+
+Lines of businesses _share the costs and risks_ of using technology with the IT branch.
+Before an IT-enabled initiative is launched, the business problem to solve with technology _is clear_, business experts _fully participate_ in the execution of the initiative in partnership with their technology colleagues, and _rapid feedback_ from end-users is not only promoted, it is _enabled_.
+
+The business vs IT paradigm is replaced with a team partnership paradigm.
+
+**2) The four (4) key metrics as identified in the [DevOps Research and Assessment (DORA) institute research](https://www.devops-research.com/research.html) for software delivery & operational performance are continuously improved**
+
+- _Lead time:_ the time code changes take to go from check-in to release in production
+- _Deployment frequency:_ the rate at which software is deployed to production or released to end users
+- _Change fail:_ the change failure rate measured by how often deployment failures occur in production that require immediate remedy
```

Comment:
Rather?

> how often deployment failures requiring immediate remedy occur in production

Reply:
I used the state of DORA's wordings for this, rather keep it that way